src/app/(profile)/(components)/ProfileList/ProfileList.tsx

```diff
@@ -2,10 +2,12 @@
 import { ProfileListItem } from '@/app/(profile)/(components)/ProfileList/ProfileListItem'
 import { useProfiles } from '@/app/(profile)/_providers/Profiles.provider'
 import { AppRoutes } from '@/utils/routes'
+import { useSession } from 'next-auth/react'
 import styles from './ProfileList.module.scss'
 
 const ProfileList = () => {
   const { filteredProfiles: profiles } = useProfiles()
+  const { data: session } = useSession()
```

Comment:
I think, you can do this inside `ProfileListItem` to avoid prop drilling. You can do the same with href 👍